library/designsystem/src/main/java/ir/composenews/designsystem/extensionFunction/Number.kt

```diff
@@ -0,0 +1,5 @@
+package ir.composenews.designsystem.extensionFunction
+
+fun Number.roundToTwoDecimalPlaces(): String {
```

Comment:
I propose to move this function into a new module named `:core:extensions`

Reply:
yes, that's a perfect idea, I agree with you, and I will move this function and its test into an extension module